examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/ISettingsProvider.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WebDemoWithEventHub
+{
+    public interface ISettingsProvider
```

Comment:
We shouldn't need this interface.

Reply:
Removed it.

---
In reply to: [426104206](https://github.com/Azure/AppConfiguration/pull/329#discussion_r426104206) [](ancestors = 426104206)